test.js

```diff
@@ -0,0 +1,18 @@
+import USBDevice from './lib/usb-device';
```

Comment:
I don't get how this works, is it the same as `#!/bin/bash`? I haven't seen this being used anywhere else.